os/binfmt/binfmt_initialize.c

```diff
@@ -81,35 +82,35 @@ void binfmt_initialize(void)
 #ifdef CONFIG_FS_BINFS
 	ret = builtin_initialize();
 	if (ret < 0) {
-		berr("ERROR: builtin_initialize failed: %d\n", ret);
+		berr("%s RAM partition : %d\n",clog_message_str[CMN_LOG_FAILED_OP], ret);
```

Comment:
it better to have builtin_initialize failed rather having as RAM partition

Reply:
I will check and change it.